test/NuGet.Clients.Tests/NuGet.CommandLine.Test/RestoreLoggingTests.cs

```diff
@@ -139,6 +140,101 @@ public async Task RestoreLogging_VerifyNU1608MessageAsync()
                 log.Message.Should().Contain("Detected package version outside of dependency constraint: x 1.0.0 requires z (= 1.0.0) but version z 2.0.0 was resolved.");
             }
         }
+        [Fact]
+        public async Task RestoreLogging_VerifyNU5036ErrorMessageAsync()
+        {
+            // Arrange
+            using (var pathContext = new SimpleTestPathContext())
+            {
+                // Set up solution, project, and packages
+                var solution = new SimpleTestSolutionContext(pathContext.SolutionRoot);
+
+                var netcoreapp1 = FrameworkConstants.CommonFrameworks.NetCoreApp10;
+
+                var projectA = SimpleTestProjectContext.CreateNETCore(
+                    "a",
+                    pathContext.SolutionRoot,
+                    netcoreapp1);
+
+                var packageX = new SimpleTestPackageContext("x", "1.0.0")
+                {
+                    Nuspec = XDocument.Parse($@"<?xml version=""1.0"" encoding=""utf-8""?>
```

Comment:
You don't really need a custom nuspec do you?
Everything will be auto-gened correct by the simple test package context.